examples/generative/md/pixelcnn.md

```diff
@@ -0,0 +1,272 @@
+    def call(self, inputs):
+        x = self.activation(inputs)
+        x = self.conv1(x)
+        x = self.pixel_conv(x)
+        x = self.conv2(x)
+        return keras.layers.add([inputs, x])
+
+
+```
+
+---
+## Build the model based on the original paper
+
+
+
+```python
+inputs = keras.Input(shape=input_shape)
+x = PixelConvLayer(mask_type="A", filters=128, kernel_size=7, padding="same")(inputs)
+
+for _ in range(n_residual_blocks):
+    x = ResidualBlock(filters=128)(x)
+
+for _ in range(2):
+    x = PixelConvLayer(
+        mask_type="B",
+        filters=128,
+        kernel_size=1,
+        strides=1,
+        activation="relu",
+        padding="valid",
+    )(x)
+
+out = keras.layers.Conv2D(
+    filters=1, kernel_size=1, strides=1, activation="sigmoid", padding="valid"
+)(x)
+
+pixel_cnn = keras.Model(inputs, out)
+adam = keras.optimizers.Adam(learning_rate=0.0001)
+pixel_cnn.compile(optimizer=adam, loss="binary_crossentropy")
+
+pixel_cnn.summary()
+pixel_cnn.fit(x=data, y=data, batch_size=64, epochs=50, validation_split=0.1)
+
+```
+
+<div class="k-default-codeblock">
+```
+Model: "model"
+_________________________________________________________________
+Layer (type)                 Output Shape              Param #   
+=================================================================
+input_1 (InputLayer)         [(None, 28, 28, 1)]       0         
+_________________________________________________________________
+pixel_conv_layer (PixelConvL (None, 28, 28, 128)       6400      
+_________________________________________________________________
+residual_block (ResidualBloc (None, 28, 28, 128)       98624     
+_________________________________________________________________
+residual_block_1 (ResidualBl (None, 28, 28, 128)       98624     
+_________________________________________________________________
+residual_block_2 (ResidualBl (None, 28, 28, 128)       98624     
+_________________________________________________________________
+residual_block_3 (ResidualBl (None, 28, 28, 128)       98624     
+_________________________________________________________________
+residual_block_4 (ResidualBl (None, 28, 28, 128)       98624     
+_________________________________________________________________
+pixel_conv_layer_6 (PixelCon (None, 28, 28, 128)       16512     
+_________________________________________________________________
+pixel_conv_layer_7 (PixelCon (None, 28, 28, 128)       16512     
+_________________________________________________________________
+conv2d_18 (Conv2D)           (None, 28, 28, 1)         129       
+=================================================================
+Total params: 532,673
+Trainable params: 532,673
+Non-trainable params: 0
+_________________________________________________________________
+Epoch 1/50
+  2/985 [..............................] - ETA: 1:02 - loss: 0.6944WARNING:tensorflow:Callbacks method `on_train_batch_end` is slow compared to the batch time. Check your callbacks.
```

Comment:
Make sure to train with the latest TF-nightly

Reply:
how do i train with tf-nightly if I'm running autogen.py add_example in colab after installing the reqs for keras using pip?

Reply:
Just `pip install tf-nightly`

Reply:
I'm still getting the same issue. I ran 
`import tensorflow as tf
tf.__version__`
'2.2.0-dev20200508'

Reply:
It only gives that output if I run the script, in the colab notebook it runs fine

Reply:
I might have figured it out, let me run it again and I'll let you know

Reply:
Try `pip install tf-nightly --upgrade`, it seems your nightly build is old. We removed this warning a couple of weeks ago.